pennylane/devices/autograd_ops.py

```diff
@@ -386,3 +386,37 @@ def DoubleExcitationMinus(phi):
     ]
 
     return np.array(U)
+
+
+def QuantumNumberPreservingOR(theta):
+    r"""Quantum number preserving four-qubit one-parameter gate.
+
+    Args:
+        theta (float): rotation angle
+    Returns:
+        array[complex]: rotation matrix
+    """
+
+    c = np.cos(theta / 2)
+    s = np.sin(theta / 2)
+
+    return np.array(
+        [
+            [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
Some of the signs are different than those in the paper, I recall that you found a typo in the original paper? If it is the case then we should document it here.

Reply:
I have documented the same now, along with the mention of the typo.